src/Components/CreateImageWizard/steps/Review/Footer/CreateDropdown.tsx

```diff
@@ -141,12 +191,12 @@ export const CreateSaveButton = ({
     const requestBody = await getBlueprintPayload();
     setIsOpen(false);
 
-    if (!process.env.IS_ON_PREMISE && !isFedoraEnv) {
-      analytics.track(`${AMPLITUDE_MODULE_NAME}-blueprintCreated`, {
-        module: AMPLITUDE_MODULE_NAME,
-        isPreview: isBeta(),
+    if (!process.env.IS_ON_PREMISE && !isFedoraEnv && requestBody) {
+      const analyticsData = createAnalytics(requestBody, packages, isBeta);
+      analytics.track(`${AMPLITUDE_MODULE_NAME} - Blueprint Created`, {
```

Comment:
Same here, changing `${AMPLITUDE_MODULE_NAME}-blueprintCreated` to `${AMPLITUDE_MODULE_NAME} - Blueprint Created` can potentially break already set up Amplitude tracking.